### DISCUSSION TITLE
Enable Hibernate on Pop OS

### DISCUSSION THREAD

**Discussion by NOOBLINUXUSER** [abskmj/notes#9] [General]
it fails to resume from hibernation. is NVIDIA to blame? even on opensource drivers. (no onboard though), it does not happen. i was enable to it before on a earlier Nvidia driver but this is weird though, i check marked 1 option in the driver settings and it worked. its not doing that anymore.
using a swapfile.

**Comment by abskmj** [comment]
I'm not sure if the NVIDIA driver is to be blamed. I had tested the steps documented on an older laptop having intel graphics though.
